clients/search-component/src/utils/hooks/useSizing.ts

```diff
@@ -27,7 +27,7 @@ const popupClass = cva(
       },
       mode: {
         search: [],
-        chat: [],
+        chat: ["md:!tv-max-h-[60vh]"],
```

Comment:
definitely down to personal preference but I think with the auto scroll, it feels better when its not as tall.